### DISCUSSION TITLE
[Combine Taskbar Icons] Icon-only (combined) if one window, Un-combined when multiple windows open

### DISCUSSION THREAD

**Discussion by Lyushen** [valinet/ExplorerPatcher#1859] [Ideas]
Hi, 
Would it be possible to uncombined if there are multiple windows opened, for example 2–3 explorers, but in case of an any other app, where is only one window - it should be only icon?
It's make more sense, when using environment. I need to see more windows on task bars where I'm working with a few, but if the window one, for example messenger or notepad++, I don't need to see whole text of their windows.
Cheers.

UPD: Browsers also should be combined in case of PWAs (for example in chrome)

**Comment by pyrates999** [comment]
I don't think the windows code itself allows for that granularly.
